lab_5_scrapper/scrapper.py

```diff
@@ -176,6 +299,16 @@ def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> No
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        self.article.title = article_soup.find('h1').text
+        date = article_soup.find(class_="blog-date").text
+        self.article.date = self.unify_date_format(date)
+        author = [''.join(article_soup.find(class_="article-author").text)]
+        if author:
+            self.article.author = author
```

Comment:
should author be present as a list type? why not as a string? lint 305 - here you put author inside []